pylibui/libui/box.py

```diff
@@ -35,33 +35,32 @@ def uiBoxAppend(box, child, stretchy):
     :return: None
     """
 
-    return clibui.uiBoxAppend(box, child, stretchy)
+    clibui.uiBoxAppend(box, child, stretchy)
 
 
-# - void uiBoxDelete(uiBox *b, uintmax_t index);
-def uiBoxDelete(*args):
+# - void uiBoxDelete(uiBox *b, int index);
```

Comment:
Sorry ! I just noticed the change of `uintmax_t` to `int` in the libui library, I thought it would be great to update the comments to match exactly the libui function signatures.